### DISCUSSION TITLE
How to update a Checkboxes field ?

### DISCUSSION THREAD

**Discussion by dsforetellix** [pycontribs/jira#1824] [Q&A]
Hi
I am using a Checkboxes field. I need to set the value to a CustomFieldOption. Here is my attempt:

```
# get the issue
issue = jira.issue(<issue_key>)
# get the custom field option with its id
new_option = jira.custom_field_option(10139)
# update
issue.update(fields={'<my_field>': [new_option]})
```

I get an exception: Object of type CustomFieldOption is not JSON serializable

I've tried with several other syntax, but I cannot make it work

**Comment by dimitarOnGithub** [comment]
Hey @dsforetellix ,

I'm not sure if this is still something on your plate, but our JIRA server also has a checkbox implementation (albeit a bit unorthodox) and what worked for us is:
```
payload = {f"{customfield_id}": [{'value': f'{value}'}]}
issue.update(fields=payload)
```
maybe that will work for you too?
